exonum_client/protobuf_loader.py

```diff
@@ -100,11 +101,25 @@ def __init__(self, client: Optional[ProtobufProviderInterface] = None):
         # Check that the client is the same as expected:
         if ProtobufLoader._entity is not None:
             if client is not None and client != ProtobufLoader._entity.client:
-                raise ValueError("Attempt to create ProtobufLoader entity with a different client")
+                err_msg = (
+                    f"Attempt to create ProtobufLoader entity with a different client. "
```

Comment:
I'd suggest to create a json string representation of objects here (create dicts with data and dump it with json library, providing some reasonable indentation value) and add newlines to the entry (be sure to use os-independent way to get newline separator) for readability reasons.

Reply:
I guess it would be resolved by `__repr__` implementation mentioned below.

Reply:
Yep.